components/org.wso2.carbon.identity.oauth.endpoint/pom.xml

```diff
@@ -322,7 +322,7 @@
                                         <limit implementation="org.jacoco.report.check.Limit">
                                             <counter>COMPLEXITY</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.54</minimum>
+                                            <minimum>0.53</minimum>
```

Comment:
Whats the reason to reduce test coverage? If the new changes are not covered lets add tests without reducing coverage

Reply:
Actually this update is a reduction of code and deprecation of a method. The updated function is covered in the tests. I think method deprecation is the main reason to reduce the code coverage. We can remove it entirely as it's a private method.

Reply:
+1 lets remove it. We dont need to deprecate